.gitlab-ci.yml

```diff
@@ -47,15 +47,23 @@ stages:
     - if [[ ${CLANG_FORMAT} && ${CI_COMMIT_REF_NAME} != next ]]; then cmake --build . --target format && git diff --submodule=diff --exit-code; fi
     - ccache -s
 
-Debug GCC:
+Debug GCC Format:
```

Comment:
@junghans what is this doing adding another test? What is the Format keyword doing?

Reply:
It is just the name of the test, I added `Format` as this test runs `clang format`.